README.md

```diff
@@ -329,3 +329,30 @@ speculate --name=my-cool-api
 ```
 
 This is useful if you are using private NPM packages which start with an `@`.
+
+
+### Mocking and node-gyp
+
+[Mock](https://github.com/rpm-software-management/mock) is a really useful tool to ensure that your RPM builds in a clean environment, without accidentally depending on something your system that you havn't added to the `require` list.
+
+While the RPM is being built by `mock`, it does not have any access to the Internet. This is good because it ensures that you get consistent builds. It also means the build won't fail, if a server outside of your control is offline. However it can cause node-gyp to fail, if it can't find the node header files on your system.
+
+Make sure your have the a build dependency on: `nodejs-devel`. This places the node header files in `/usr/include/node`:
```

Comment:
It's worth noting that it's `nodejs-devel` for nodesource but is actually included by default in other RPMs. So might need to clarify that this is an example for nodesource?

Reply:
Ah! I didn't realise that. I will get it updated.